packages/react-components/react-nav-preview/src/components/NavDrawerBody/NavDrawerBody.types.ts

```diff
@@ -1,17 +1,14 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { DrawerBodySlots, DrawerBodyState } from '@fluentui/react-drawer';
+import type { ComponentProps, ComponentState } from '@fluentui/react-utilities';
 
-export type NavDrawerBodySlots = {
-  root: Slot<'div'>;
-};
+export type NavDrawerBodySlots = ComponentProps<DrawerBodySlots>;
 
 /**
  * NavDrawerBody Props
  */
-export type NavDrawerBodyProps = ComponentProps<NavDrawerBodySlots> & {};
+export type NavDrawerBodyProps = ComponentProps<DrawerBodySlots>;
```

Comment:
I think since we aren't adding any props, we could just use directly Drawer's props
```suggestion
export type NavDrawerBodyProps = DrawerBodyProps;
```

if the idea is to rename the type to NavDrawerBodySlots, then it should be like this
```suggestion
export type NavDrawerBodyProps = ComponentProps<NavDrawerBodySlots>;
```